apps/mobile/tailwind.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         background: 'hsl(var(--background) / <alpha-value>)',
         midground: 'hsl(var(--midground) / <alpha-value>)',
         foreground: 'hsl(var(--foreground) / <alpha-value>)',
+        selected: 'hsl(var(--selected) / <alpha-value>)',
```

Comment:
I'm baffled why these changes show in my branch and in difference to the base branch of `drawer-experience`? I didn't make them, but I do think these are things that were going forward 🤷🏼‍♂️